### PR TITLE
feat(booking): add airline-initiated changes

### PR DESCRIPTION
**Pull request by matthewp5**
## 🔍 Overview

This PR adds airline-initiated changes endpoint functions to the javascript library.

## 🧑‍💻 What has been done?

- Updated `AirlineInitiatedChange` interface and moved to separate folder
- Added endpoint functions
- Added mock objects
- Added tests

## 🦮 Reviewer guide

I'd recommend reviewing this PR commit-by-commit.

### CODE REVIEW

**Review by go1t — APPROVED** [review]

File: src/booking/AirlineInitiatedChanges/AirlineInitiatedChanges.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+   * the airline-initiated change
+   *
+   * @link https://duffel.com/docs/api/v1/airline-initiated-changes/accept-airline-initiated-changes
+   */
+  public accept = async (
+    id: AirlineInitiatedChange['id']
+  ): Promise<DuffelResponse<AirlineInitiatedChange>> =>
+    this.request({
+      method: 'POST',
+      path: `${this.path}/${id}`,
```
Comment (go1t) [inline src/booking/AirlineInitiatedChanges/AirlineInitiatedChanges.ts:76]: Shouldn't this be
```suggestion
      path: `${this.path}/${id}/actions/accept`
```

Comment (matthewp5) [reply src/booking/AirlineInitiatedChanges/AirlineInitiatedChanges.ts:76]: Ah yes good spot, will update

File: src/booking/AirlineInitiatedChanges/AirlineInitiatedChanges.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+   * respectively.
+   *
+   * @link https://duffel.com/docs/api/v1/airline-initiated-changes/update-airline-initiated-changes
+   */
+  public update = async (
+    id: AirlineInitiatedChange['id'],
+    action_taken: AirlineInitiatedChangeAvailableAction
+  ): Promise<DuffelResponse<AirlineInitiatedChange>> =>
+    this.request({
+      method: 'POST',
```
Comment (go1t) [inline src/booking/AirlineInitiatedChanges/AirlineInitiatedChanges.ts:58]: This should be `'PATCH'` as well haha

**Review by go1t — APPROVED** [review]
LGTM!

**Pull request merged**